test/workbox-precaching/sw/test-matchPrecache.mjs

```diff
@@ -0,0 +1,56 @@
+/*
+  Copyright 2019 Google LLC
+
+  Use of this source code is governed by an MIT-style
+  license that can be found in the LICENSE file or at
+  https://opensource.org/licenses/MIT.
+*/
+
+import {matchPrecache} from 'workbox-precaching/matchPrecache.mjs';
+import {precache} from 'workbox-precaching/precache.mjs';
+
+describe(`matchPrecache()`, function() {
```

Comment:
Should this, instead, be testing the `matchPrecache()` method on `PrecacheController`, and then the test for the plain export can just ensure a default controller is created and its `matchPrecache()` method called?

Reply:
This matches what we're doing with other tests modules that follow a similar pattern (c.f. [`test-getCacheKeyForURL.mjs`](https://github.com/GoogleChrome/workbox/blob/f23924a988a3e321dbcd9e3781f07b1de5dce916/test/workbox-precaching/sw/test-getCacheKeyForURL.mjs)).

So... maybe we can keep this test implemented as-is and if we want a different approach, update all of the relevant tests to follow that model?